examples/platform/linux/CommissioneeShellCommands.h

```diff
@@ -0,0 +1,33 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
```

Comment:
```suggestion
 *    Copyright (c) 2021 Project CHIP Authors
```
and similar elsewhere in this PR.

Reply:
addressed in latest changes